yaml/training_prod/ingress_de.yaml

```diff
@@ -1,18 +1,13 @@
 apiVersion: extensions/v1beta1
 kind: Ingress
 metadata:
-  annotations:
-    field.cattle.io/publicEndpoints: '[{"addresses":["2a06:c00::37e9"],"port":443,"protocol":"HTTPS","serviceName":"pitc-ansible-training-prod:latest-ansible-puzzle","ingressName":"pitc-ansible-training-prod:latest-ansible-puzzle","hostname":"ansible.puzzle-itc.de","path":"/","allNodes":true}]'
-    kubernetes.io/ingress.class: nginx-public
-    kubernetes.io/tls-acme: "true"
-    meta.helm.sh/release-name: latest
-    meta.helm.sh/release-namespace: pitc-ansible-training-prod
   labels:
     app.kubernetes.io/instance: latest
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/name: ansible-puzzle
     app.kubernetes.io/version: 1.0.0
-    helm.sh/chart: acend-training-chart-0.1.2
+    helm.sh/chart: acend-training-chart-0.1.4
```

Comment:
You shouldn't add helm labels if the resource is not managed by helm. Additionally, these labels would be set by helm itself.

Reply:
removed